packages/vaex-core/vaex/__init__.py

```diff
@@ -135,10 +125,18 @@ def open(path, convert=False, progress=None, shuffle=False, fs_options={}, fs=No
     Files are by default cached in $HOME/.vaex/file-cache/(s3|gs) such that successive access
     is as fast as native disk access.
 
+    Amazon AWS S3 options:
+
     The following common fs_options are used for S3 access:
 
-     * anon: Use anonymous access or not (false by default). (Allowed values are: true,True,1,false,False,0)
-     * cache: Use the disk cache or not, only set to false if the data should be accessed once. (Allowed values are: true,True,1,false,False,0)
+     * `anon`: Use anonymous access or not (false by default). (Allowed values are: true,True,1,false,False,0)
+     * `anonymous` - Alias for `anon`
+     * `cache`: Use the disk cache or not, only set to false if the data should be accessed once. (Allowed values are: true,True,1,false,False,0)
+     * `access_key` - AWS access key, if not provided will use the standard env vars, or the `~/.aws/credentials` file
+     * `secret_key` - AWS secret key, similar to `access_key`
+     * `profile` - If multiple profiles are present in `~/.aws/credentials`, pick this one instead of 'default', see https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html
+     * `region` - AWS Region, e.g. 'us-east-1`, will be determined automatically if not provided.
+     * `endpoint_override` - URL/ip to connect to, instead of AWS, e.g. 'localhost:9000' for minio
```

Comment:
Nice, i like this change

Reply:
Stemmed from sphinx throwing warnings about indentation in the arg list.